api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+import com.datastrato.gravitino.annotation.Evolving;
+
+/** The interface of a privilege. */
+@Evolving
+public interface Privilege {
+
+  /** @return The generic name of the privilege. */
+  Name name();
+
+  /** @return A readable string representation for the privilege. */
+  String simpleString();
+
+  /** The name of this privilege. */
+  enum Name {
+    /** The privilege of load a metalake. */
+    LOAD_METALAKE
```

Comment:
Do we need a metalake level privilege? I think every user added into the metalake should have the permission to load metalake, what we should expose is catalog/schema/xxx, it should be enough.

Reply:
I removed here.